ui/apps/platform/src/utils/getFragment.js

```diff
@@ -0,0 +1,12 @@
+import queryString from 'qs';
+
+export function parseFragment(location) {
```

Comment:
To save a step for when we rewrite LoginPage as TypeScript, can we write this util file now?

```ts
export function parseFragment(location: Location): Record<string, string> {
```